File: compose-app/shared/src/commonMain/kotlin/com/sales/app/domain/model/Address.kt
package com.sales.app.domain.model

data class Address(
    val id: Int,
    val partyId: Int,
    val accountId: Int,
    val line1: String,
    val line2: String?,
    val city: String,
    val district: String?,
    val state: String,
    val pincode: String,
    val country: String,
    val latitude: Double?,
    val longitude: Double?
)

File: backend/app/Models/Address.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Address extends Model
{
    use SoftDeletes, \App\Traits\HasLog;

    protected $fillable = [
        'party_id',
        'line1',
        'line2',
        'line2',
        'place',
        'district',
        'state',
        'country',
        'pincode',
        'latitude',
        'longitude',
        'account_id',
        'log_id',
    ];

    public function party()
    {
        return $this->belongsTo(Party::class);
    }

    public function account()
    {
        return $this->belongsTo(Account::class);
    }
}
